### ISSUE TITLE
Logout Branch 

### ISSUE THREAD

**Issue by cemakkok** [openstf/stf#437]
Hi Sorccu,

Please open a new branch and add logout button, i cannot develop it because of websocket or you can write code piece for logout function and i can add my project ?

**Comment by sorccu** [comment]
That's not how open source works... I don't work for you. You can make a feature request and **maybe** someone will look into it at some point *if and when they want*.

**Comment by cemakkok** [comment]
Yes, you are right Sorccu but i said **please**.

**Comment by haluklevent** [comment]
yeah it is a necessity, I have read all issues about logout and timeout, anyone resolve that. 
you may direct us simo

**Comment by gunta** [comment]
@cemakkok You should feel free to open issues, but you shouldn't ask someone in particular since that is just asking to work for free. Someone else might want to implement it also.

Since this is open source, everyone works on their spare free time and the issues they care most first. 

If you really need it ASAP you can try posting a bounty [here](https://www.bountysource.com/issues/38007749-logout-branch) and someone might give it a chance.
